account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -375,7 +360,13 @@ def _get_real_xml_name(self, template):
         (name, module) = external_id.split(".")
         return "%s.%d_%s" % (name, self.company_id.id, module)
 
-    @api.multi
+    @tools.ormcache("templates")
+    def find_taxes_by_templates(self, templates):
+        tax_ids = []
+        for tax in templates:
+            tax_ids.append(self.find_tax_by_templates(tax))
+        return self.env["account.tax"].browse(tax_ids)
+
     @tools.ormcache("templates")
     def find_tax_by_templates(self, templates):
```

Comment:
Same question as above... Well, just take a look at the whole file, it's full of these things.